src/lib.rs

```diff
@@ -99,7 +99,7 @@ pub enum KVMiMsr {
 pub struct KVMiEvent {
     pub vcpu: u16,
     pub ev_type: KVMiEventType,
-    ffi_event: *mut kvmi_dom_event,
+    pub ffi_event: *mut kvmi_dom_event,
```

Comment:
I like that you have removed the `kvmi_dom_event` for this PR, but you forgot this line here ;)